immersion/kubeflow_pipelines/cicd/solutions/lab-03_vertex.ipynb

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
<p>for the build to succeed I believe you need to add <code>roles/aiplatform.user</code>  to the compute service account. Else I get an error with missing <code>aiplatform.pipelineJobs.create</code>  permission. See also https://cloud.google.com/vertex-ai/docs/general/access-control</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/117/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Added this as a remark if the user experience issues.</p>